pcap-mode.el

```diff
@@ -72,12 +72,22 @@
 ;;;###autoload
 (defcustom pcap-mode-tshark-filter ""
   "Filter to apply to tshark invocations."
-  :type 'string)
+  :type 'string
```

Comment:
Have updated the changelog, but forgot to add signed off lines, sorry (aren't my changes automatically covered by the GPL anyway?).